policy-controller/k8s/index/src/index.rs

```diff
@@ -1142,6 +1142,10 @@ impl PolicyIndex {
                     }
                 }
                 authorization_policy::Target::Namespace => {}
+                authorization_policy::Target::HttpRoute(_) => {
+                    // Skip the policy if it targets an HttpRoute.
+                    continue;
```

Comment:
This comment is a little redundant. It might be helpful, though, _why_ it's okay to skip the policy. I.e., explain where these policies are going to be handled.